spec/moonshot/plugins/ssh_spec.rb

```diff
@@ -0,0 +1,50 @@
+describe Moonshot::Asg::SSH do
```

Comment:
Can we follow the class file path here on the spec `spec/moonshot/plugins/rotate_asg_instances/ssh_spec.rb `